packages/marshal/src/passStyleOf.js

```diff
@@ -236,17 +236,23 @@ export { assertIface };
  *   [Symbol.toStringTag]: string,
  *   toString: () => void }} val the value to verify
  * @param {Checker} [check]
+ * @param {any} original for better diagnostics
  * @returns {boolean}
  */
-const checkRemotableProto = (val, check = x => x) => {
+const checkRemotableProto = (val, check = x => x, original = undefined) => {
   if (
     !(
       check(
         typeof val === 'object',
         X`cannot serialize non-objects like ${val}`,
       ) &&
       check(!Array.isArray(val), X`Arrays cannot be pass-by-remote`) &&
-      check(val !== null, X`null cannot be pass-by-remote`)
+      check(val !== null, X`null cannot be pass-by-remote`) &&
+      check(
+        // @ts-ignore
```

Comment:
Please write the reason for the ts-ignore on the same line:
```suggestion
        // @ts-ignore - this is the reason we needed to ignore
```

Reply:
Done